Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+import Banner from'./CardBanner';
+import CardContent from './CardContent';
+
+
+
+const CardContainer = () => {
+    return (
+       <div className="bottom">
+        <Banner />
```

Comment:
You should still follow html spacing here. `<Banner />` is a child of the `div`.